bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/tac/TacView.scala

```diff
@@ -188,7 +189,7 @@ class TacView(loc: Locale) extends BorderPanel with BoundView[ProposalClass] { v
       override def refresh(m:Option[Option[SubmissionDecision]]) {
```

Comment:
Everything looks good here :+1:.  The comment Art was referring to in his IM was one I made about the `Option[Option[SubmissionDecision]]` parameter which looks suspicious.  Perhaps it should have been joined at a higher level but of course that isn't the point of this PR.  At least you did get rid of the no-op `headOption` below.